src/client/download.ts

```diff
@@ -37,12 +44,26 @@ export class Download extends ChannelOwner<channels.DownloadChannel, channels.Do
   }
 
   async path(): Promise<string | null> {
+    if (this._browser && this._browser._isRemote)
```

Comment:
This is very weird contract. Should we remove this method altogether from the (client) API?

Reply:
But it works in the non-remote case, and we use the same client code there.